crates/subspace-core-primitives/src/objects.rs

```diff
@@ -43,6 +43,7 @@ pub enum BlockObject {
     #[cfg_attr(feature = "serde", serde(rename_all = "camelCase"))]
     V0 {
         /// Object hash
+        #[cfg_attr(feature = "serde", serde(with = "hex"))]
```

Comment:
I always used `serde(with = "hex::serde")`, didn't notice there were re-exports there. Probably should change all occurrences to this shorter form.

Reply:
Someone can do this in another PR, is it worth opening a ticket or making a TODO?

Reply:
It takes a single search/replace in the project to do this, faster than submitting a comment. Creating TODO or issue for this is even more bureaucracy. Either here or in a separate PR I don't care, but I see no point in creating tickets for trivial issues.